utils/src/lib.rs

```diff
@@ -404,10 +405,36 @@ async fn wait_for_transaction_to_be_mined<P: JsonRpcClient>(
         sleep(interval).await;
     }
 
-    tracing::error!("contraact call {hash:?}: not mined after {retries} retries");
+    tracing::error!("contract call {hash:?}: not mined after {retries} retries");
     false
 }
 
+pub async fn wait_for_anvil_endpoints<P: JsonRpcClient>(provider: &Provider<P>) {
```

Comment:
Should we make this part of `spawn`, so that `AnvilOptions::spawn()` doesn't return until Anvil is ready? That way we can't forget to call it in some new test or something

Reply:
I agree. Done in f05cad023f3daea69388b63d2991f9ffe341b823.